src/test/ui/fmt/format-string-error-2.stderr

```diff
@@ -1,3 +1,11 @@
+error: incorrect unicode escape sequence
+  --> $DIR/format-string-error-2.rs:77:20
+   |
+LL |     println!("/x7B}/u8 {", 1);
```

Comment:
Wait ... the .rs file source line said `println!("\x7B}\u8 {", 1);` (with backslashes), why are there forward slashes in the output here?

Reply:
@zackmdavis can't find the PR, but UI tests normalize backslashes to forward slashes for Windows' sake to avoid further differences across platforms when running tests. It always makes me recoil for a second until I remember that that's a thing.